1_Intro_HyperSpy-LumiSpy-eXSpy.ipynb

```diff
@@ -428,7 +426,9 @@
     "\n",
     "The signal beyond 800 nm goes to negative values, so lets remove the last three pixels from every spectrum (using signal indexation) and replace the original signal.\n",
     "\n",
-    "*NOTE: Indexation operates on pixel in the signal dimension if the given number is an integer and on the calibrated (wavelength axis) if a float value is used as index.*"
+    "*NOTE: Indexation operates on pixel in the signal dimension if the given number is an integer and on the calibrated (wavelength axis) if a float value is used as index.*\n",
+    "\n",
+    "*Caution: Only run this cell once, since each consecutive rum will remove another three pixels. If you do not want that you can dedicate a diffent variable for the corrected signal.*"
```

Comment:
```suggestion
    "*Caution: Only run this cell once, since each consecutive run will remove another three pixels. If you do not want that you can dedicate a diffent variable for the corrected signal.*"
```